core/apps/ame-e2e/src/fixtures/time-series-entity.txt

```diff
@@ -0,0 +1,20 @@
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.openmanufacturing.digitaltwin:1.0.0#>.
+
+:AspectDefault a bamm:Aspect;
+    bamm:properties (:property1);
+    bamm:operations ();
+    bamm:events ().
+:property1 a bamm:Property;
+    bamm:characteristic :Characteristic1.
+:Characteristic1 a bamm:Characteristic;
+    bamm:dataType :Entity1.
+:Entity1 a bamm:Entity;
+    bamm:properties ([bamm:extends bamm-e:value]);
```

Comment:
Same here. I am not sure what you want to use this Aspect model for, but the definition for this property is not valid. The characteristic is missing.

Reply:
In this case we test exactly this case, that we can load it but it is invalid.